draft-ietf-dnsop-serve-stale.md

```diff
@@ -473,16 +465,18 @@ records alive even after their authorities have gone away.  This
 potentially makes that easier, although without introducing a new
 risk.
 
-In {{CloudStrife}} it was demonstrated how stale DNS data, namely
+In {{CloudStrife}}, it was demonstrated how stale DNS data, namely
 hostnames pointing to addresses that are no longer in use by the owner
 of the name, can be used to co-opt security such as to get
 domain-validated certificates fraudulently issued to an attacker.
-While this RFC does not create a new vulnerability in this area, it
+While this document does not create a new vulnerability in this area, it
 does potentially enlarge the window in which such an attack could be
-made.  An obvious mitigation is that not only should a certificate
-authority not use a resolver that has this feature enabled, it should
-probably not use a caching resolver at all and instead fully look up
-each name freshly from the root.
+made.  A proposed mitigation is that certificate
```

Comment:
obvious -> proposed is fine. For the rest, the existing text reads better to me.

Reply:
More direct wording:

"A proposed mitigation is that certificate authorities should fully look up each name starting at the DNS root for every name lookup. Alternatively, CAs should use a resolver that is not serving stale data."

Reply:
I disagree with the second sentence because there is no way for a resolver's client to know if the data they are getting is served stale. Even if the CA sets up its own resolver and does not turn on stale serving, the resolver software itself might be doing that without warning. At best, you could say "Alternatively, a CA could use a resolver that it believes is not serving stale data."

Reply:
That "the resolver software itself might be doing that without warning" does not invalidate the meaning of the sentence.